docs/source/release_notes.rst

```diff
@@ -37,6 +37,7 @@ Release Notes
         * Added labeling to ``graph_confusion_matrix`` :pr:`1632`
         * Rerunning search for ``AutoMLSearch`` results in a message thrown rather than failing the search, and removed ``has_searched`` property :pr:`1647`
         * Changed tuner class to allow and ignore single parameter values as input :pr:`1686`
+        * Capped LightGBM version limit to remove bug in docs :pr:`1711`
```

Comment:
Any idea why this is only reproducible in the docs?

Reply:
Unsure why, but when I ran locally through Jupyter, I didn't run into this bug, even when switching versions for LightGBM. I can look a little more tomorrow and see if there's a reason though.

Reply:
Maybe looking at the entire stack trace will be helpful?

Reply:
Updated the more complete stacktrace in the description box. I suspect something in nbsphinx might be causing the bug to appear, but I'm unable to repro it in Jupyter.